proposals/nnnn-screenmanager-push-pop.md

```diff
@@ -0,0 +1,129 @@
+# Screen Manager Template Management
+
+* Proposal: [SDL-NNNN](NNNN-screenmanager-push-pop.md)
+* Author: [Joel Fischer](https://github.com/joeljfischer)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS / Java Suite / Web]
```

Comment:
Can you please specify which `Web` component(s) will be impacted?  For example, SHAID, Developer Portal, SDL Server, etc.